modules/polls/components/PollVoteResultsCompact.tsx

```diff
@@ -0,0 +1,97 @@
+/** @jsx jsx */
+import { Box, Text, jsx } from 'theme-ui';
+import { PollVoteHistory } from '../types/pollVoteHistory';
+
+export function PollVoteResultsCompact({vote }: {vote: PollVoteHistory}): React.ReactElement {
+  const styles = {
+    titles: {
+      display: 'flex',
+      justifyContent: 'space-between'
+    },
+    progressBarWrapper: {
+      display: 'flex',
+      borderRadius: '4px',
+      overflow: 'hidden',
+      marginTop: 2,
+      marginBottom: 2
+    },
+    yesBar: {
+      backgroundColor: 'primary',
+      width: '80%',
+      height: '4px'
+    },
+    noBar: {
+      backgroundColor: 'notice',
+      width: '10%',
+      height: '4px'
+    },
+    abstainBar: {
+      backgroundColor: 'secondary',
+      width: '10%',
+      height: '4px'
+    },
+    percentagesWrapper: {
+      display: 'flex',
+      justifyContent: 'space-between'
+    },
+    yesPercentage: {
+      color: 'primary'
+    },
+    noPercentage: {
+      color: 'notice'
+    },
+    abstainPercentage: {
+      color: 'secondaryEmphasis'
+    },
+    
+  };
+
+  return (
+    <Box>
+      <Box sx={styles.titles}>
+        <Box>
+          <Text
+            variant="secondary"
+            color="onSecondary"
+            sx={{ textTransform: 'uppercase', fontSize: 1, fontWeight: 'semiBold', ml: 1, mr: 1}}
+            as="p"
+          >
+            Yes
+          </Text>
+        </Box>
+        <Box>
+          <Text
+            variant="secondary"
+            color="onSecondary"
+            sx={{ textTransform: 'uppercase', fontSize: 1, fontWeight: 'semiBold', ml: 1, mr: 1}}
+            as="p"
+          >
+            Abstain
+          </Text>
+        </Box>
+        <Box>
+          <Text
+            variant="secondary"
+            color="onSecondary"
+            sx={{ textTransform: 'uppercase', fontSize: 1, fontWeight: 'semiBold', ml: 1, mr: 1}}
+            as="p"
+          >
+            No
+          </Text>
+        </Box>
+      </Box>
```

Comment:
Feels off to have vote options hard-coded, we should be able to get these dynamically for each poll. Occasionally there are different options than these three which would make this component not compatible for those.

Reply:
Yep, this component is not finished, because the design really doesn't fit real requirements. 

Design is limited to this 3 options but polls can have many, then this component should be re-designed